rbi/yast/scr.rbi

```diff
@@ -0,0 +1,20 @@
+# typed: strong
+
+module Yast::SCR
+  sig do
+    params(
+      scr_path: T.any(String, Yast::Path),
+      arg1: T.untyped,
+      arg2: T.untyped
+    ).returns(T.untyped)
+  end
+  def self.Execute(scr_path, arg1 = T.unsafe(nil), arg2 = T.unsafe(nil)); end
+
+  sig do
+    params(
+      scr_path: T.any(String, Yast::Path),
+      arg1: T.untyped
+    ).returns(T.untyped)
+  end
+  def self.Read(scr_path, arg1 = T.unsafe(nil)); end
```

Comment:
Write is missing due to WIP or is there other reason?

Reply:
Yup, just the minimal viable prototype. SCR.Write is pretty high in the TODO list.